content/posts/why-we-choose-headless-components-for-cunningham/index.md

```diff
@@ -0,0 +1,245 @@
+---
+title: "Why we choose headless components for Cunningham"
+date: 2023-02-03T17:16:58Z
+description:
+  "France Université Numérique has developed several applications and recently decided to create
+  their own design system, Cunningham, to provide a cohesive experience for users and streamline the
+  development process. They considered three approaches for constructing the component library:
+  utilizing a component library, developing components from scratch, and implementing headless
+  components. After evaluating the pros and cons of each approach, they decided to use headless
+  components, which offer a balance between customization and development efforts."
+categories: ["frontend", "cunningham"]
+draft: false
+authors:
+  - "nathanvasse"
+  - "jbpenrath"
+---
+
+At France Université Numérique, we are developing several applications such as
+[Richie](https://github.com/openfun/richie), [Marsha](https://github.com/openfun/marsha),
+[Potsie](https://github.com/openfun/potsie), etc. To avoid writing duplicate and inconsistent
+components, we recently decided to create our own design system called
+[Cunningham](https://github.com/openfun/cunningham). We believe Cunningham will provide a more
```

Comment:
```suggestion
[Cunningham](https://github.com/openfun/cunningham). We believe Cunningham will provides a more
```

Reply:
It's future simple so we are using will + base form of the verb.